doc/install_windows.txt

```diff
@@ -1,633 +1,125 @@
+    OpenCL support is still minimal for now.
 
-- Edit ``pythonxy\mingw\msys\1.0\msys.bat`` (e.g. in Wordpad) and add as first
-  line ``set HOME=%USERPROFILE%``. Then create an easily accessible shortcut
-  (e.g. on your desktop) to this file, run it and within the MSYS
-  console, run the MSYS post-install script:
+Install CUDA drivers
+^^^^^^^^^^^^^^^^^^^^
 
-    .. code-block:: bash
+Follow `this link <https://developer.nvidia.com/cuda-downloads>`__
+to install the CUDA driver and the CUDA Toolkit.
 
-        /postinstall/pi.sh
+You must reboot the computer after the driver installation.
 
-  It will ask for your MinGW installation directory (e.g.
-  ``c:/pythonxy/mingw``; note the forward slashes).
+Installation of Theano and libgpuarray
+======================================
 
-Once you have a working MinGW/MSYS shell environment, you can go on as
-follows:
+Stable Installation
+-------------------
 
-  a) Install ``wget`` by running the setup program you can download on the
-     `wget website <http://gnuwin32.sourceforge.net/packages/wget.htm>`__.
-     Note that this setup does not add ``wget`` into the system PATH, so you
-     will need to modify the ``PATH`` environment variable accordingly (either in
-     Windows or in a ``.profile`` startup file in your MinGW home). Once this is done,
-     type ``wget --version`` in a MinGW shell to verify that it is running
-     properly. Note also that if you are behind a proxy, you should set up your
-     ``HTTP_PROXY`` environment variable, or use a custom ``wgetrc`` config file
-     for wget to be able to download files.
+With ``conda``
+^^^^^^^^^^^^^^
 
-  b) Unzip OpenBLAS and, in a MinGW shell, go into the corresponding directory.
+If you use conda, you can directly install both theano and pygpu. Libgpuarray
+will be automatically installed as a dependency.
 
-  c) Compile OpenBLAS with:
+.. code-block:: bash
 
-    .. code-block:: bash
+    conda install theano pygpu
 
-      quickbuild.win32 1>log.txt 2>err.txt
+With ``pip``
+^^^^^^^^^^^^
 
-    (use ``quickbuild.win64`` for 64-bit Windows).
-    Compilation can take a while, so now is a good time to take a break.
-    When it is done, you should have ``libopenblas.dll`` in your OpenBLAS
-    folder. If that is not the case, check the ``err.txt`` log for build errors.
+If you use pip, you have to install Theano and libgpuarray separately.
 
-  d) Make sure that ``libopenblas.dll`` is in a folder that is in your ``PATH``.
+theano
+::::::
 
-  e) Modify your .theanorc (or .theanorc.txt) with
-     ``ldflags = -LX:\\YYY\\ZZZ -lopenblas`` where ``X:\\YYY\\ZZZ`` is the path
-     to the folder containing ``libopenblas.dll``.
-     This setting can also be changed in Python for testing purpose (in which
-     case it will remain only for the duration of your Python session):
+.. comment: Installation instructions for theano for stable version.
+.. include:: install_generic.inc
+    :start-line: 11
+    :end-line: 29
```

Comment:
I have not yet found how to do that with anchors, but now I use `start-after` and `end-before` instead of `start-line` and `end-line`.

`start-after` and `end-before` take texts. I have added comments that look like anchors, and I use them as delimiters around texts to include.

I can't use anchors inside `install_generic.inc` because this file is included in many files, then anchors are duplicated and warnings are raised.